global_costmap:
  start_time: 5
  padding: 4 # number of grid cells to pad the c_scpace with